README.md

```diff
@@ -23,3 +23,32 @@ If you want to quickly run a docker container of this app then you can:
 ## Docker Container Usage
 1. docker start sniffr-be...
 2. docker stop sniffr-be...
+
+
+# Backend Routes
+## User Routes
+#### `/login` 
+
+Success example (*with my correct password cesnored to xxxxxx*):  
+![login success example](extra\readme_images\user_success.PNG)  
```

Comment:
I suggest we we stick to lowercase file extensions wherever possible.
Reason: Windows is not case sensitive. If there is a case mismatch anywhere, it won't be detected locally but deployment may fail. Git may also not register a filename case mismatch.

Reply:
Good point -- that was me being lazy with copy paste.